README.md

```diff
@@ -146,4 +147,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 
 This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
 
+If you do contribute, please run ```npm run lint``` and ```npm test``` before you submit your code.
```

Comment:
I'm not sure anyone ever will notice this sentence as it is at the very end below list of contributors, better add a section above called `## Development`

Reply:
Done.

Reply:
Done.